### PR TITLE
Fix wrongs Language Names in Working Groups page.

### PR DESCRIPTION
**Pull request by pierreneter**
Edit language names.
Related to: 
* https://github.com/nodejs/CTC/pull/168
* https://nodejs.org/en/about/working-groups/

### CODE REVIEW

File: locale/en/about/working-groups.md [hunk 1]
```diff
@@ -113,27 +113,27 @@ Each language community maintains its own membership.
...
+* [nodejs-ms - Malay (بهاس ملايو‎)](https://github.com/nodejs/nodejs-ms)
 * [nodejs-nl - Dutch (Nederlands)](https://github.com/nodejs/nodejs-nl)
 * [nodejs-no - Norwegian (Norsk)](https://github.com/nodejs/nodejs-no)
 * [nodejs-pl - Polish (Język Polski)](https://github.com/nodejs/nodejs-pl)
 * [nodejs-pt - Portuguese (Português)](https://github.com/nodejs/nodejs-pt)
 * [nodejs-ro - Romanian (Română)](https://github.com/nodejs/nodejs-ro)
 * [nodejs-ru - Russian (Русский)](https://github.com/nodejs/nodejs-ru)
 * [nodejs-sv - Swedish (Svenska)](https://github.com/nodejs/nodejs-sv)
-* [nodejs-ta - Tamil (தமிழ்)](https://github.com/nodejs/nodejs-ta)
+* [nodejs-ta - Tamil (தமிழர்)](https://github.com/nodejs/nodejs-ta)
```
Comment (thefourtheye) [inline locale/en/about/working-groups.md]: This is not correct. தமிழர் refers to the people who speak the language தமிழ்.

**Review by thefourtheye — DISMISSED** [review]

File: locale/en/about/working-groups.md (hunk 1 above)
Comment (pierreneter) [reply locale/en/about/working-groups.md]: @thefourtheye fixed.

**Review by digitalinfinity — APPROVED** [review]
Hindi and Arabic changes look good to me

**Comment by fhemberger** [comment]
Ok, no further comments/objections, so I'm merging this.
Thanks @pierreneter for finding this!

**Pull request merged**